dev/com.ibm.ws.webcontainer.servlet.3.1_fat/bnd.bnd

```diff
@@ -59,10 +56,16 @@ tested.features:\
     appsecurity-3.0,\
     appsecurity-4.0,\
     servlet-6.0,\
-    expressionlanguage-5.0,\
+    expressionLanguage-5.0,\
     pages-3.1,\
     cdi-4.0,\
-    appsecurity-5.0
+    appsecurity-5.0,\
+    noship-1.0,\
+    servlet-6.1,\
+    expressionLanguage-6.0,\
+    pages-4.0,\
+    cdi-4.1,\
+    appsecurity-6.0
```

Comment:
[appSecurity-x.0](https://openliberty.io/docs/latest/reference/feature/appSecurity-5.0.html)  is all lower case

However, I don't think case matters here because  of `.trim().toLowerCase()` is used with the feature processing here - https://github.com/OpenLiberty/open-liberty/blob/4c5ff746c397155eccb87365b7cacec134966e48/dev/fattest.simplicity/src/componenttest/depchain/FeatureDependencyProcessor.java

Reply:
Let me use lowercase here then for expressionlanguage and then I'll clean it up in some other buckets in a follow on PR. The FAT logs do list lowercase so makes sense.